### ISSUE TITLE
Natural Armor for species should have a dex maximum of 4

### ISSUE THREAD

**Issue by itamarcu** [mlenser/kryx-rpg-issues#588]
Some species have Natural Armor.  For example, the [Thri-Kreen](https://www.kryxrpg.com/character/species/thri-kreen):
> Natural Armor. You have a tough carapace. When you aren’t wearing armor, your Defense is 12 + half your proficiency bonus + your Dexterity. You can use your natural armor to determine your Defense if the armor you wear would leave you with a lower Defense. A shield’s benefits apply as normal while you use your natural armor.

Those species grant natural armor of 12 + half prof + Dex.  This caps at a possible maximum of 17 + half prof.

Compare this to other options:

- [Frost Armor](https://www.kryxrpg.com/spells/frost-armor) (+several other spells) - 12 + half prof + Dex (maximum 4), so cap is 16 + half prof
- [Leather Armor](https://www.kryxrpg.com/equipment/armor) - 11 + half prof + Dex, so cap is 16 + half prof
- Hide armor (same link) - 12 + half prof + Dex (maximum 4), again, cap is 16 + half prof
- Heavy armors (same link) - all cap at a total maximum of 17 + half prof, plus sometimes soak, but have weight, Str requirement, proficiency requirement, and high cost.
- [Acolyte](https://www.kryxrpg.com/character/classes/acolyte)'s Armor of Faith - 9 + half prof + Dex + Wis (maximum 3), which caps at 17 + half prof, but prevents using a shield.

Overall - natural armor, for a dex-focused character, is the best defense.  For characters without heavy armor proficiency that can still hold a shield, it is also the best defense.  The spells (or feats) that grant magical armor are not as good as natural armor.

I think this is problematic.  For example, a player who wants to play a dex-based Naturalist (who has proficiency with up to hide armor, and shields) will be mechanically encouraged to play a species with natural armor, even though it doesn't really make sense flavorfully for a natural species to have better armor than what a spell can grant, or what money can buy.

Also important to mention: the [Spreadsheet](https://docs.google.com/spreadsheets/d/1LMmA8rGcXLxNz_EnH3SrD5R70UaQaYupdve5nmKtBcY/edit#gid=0) says that this feature is worth 1 value point, because:
> Does not make a difference normally for classes with higher than light armor, and is equivalent to mage armor.

Which isn't true without adding a Dex maximum of 4.

So, TL;DR - my suggestion is to add the "(maximum 4)" to the Defense of this feature in species.

**Comment by mlenser** [comment]
Even if this was added, a dex-based Rogue would still get +1 AC until level 7 by choosing one of the 4 species with natural armor.

**Comment by itamarcu** [comment]
You're right, though that's a separate issue.  Adding the cap is mostly important for high dex or high level characters.

For Rogues and such, I don't know how useful this feat is supposed to be, but at least it isn't unique - the Rogue could get the same benefit by getting a spell such as mage armor, or a feat for better armors.  Perhaps there could be a feat like [Persistent Armor](https://www.kryxrpg.com/feats/persistent-armor) that gives you similar permanent armor, but with flavor that doesn't require you to be a spellcaster.

**Comment by mlenser** [comment]
Dragonborn, Ssurran, Thri-Kreen, and Warforged armor like features adjusted to limit their Dexterity
